modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/120_data_streams_stats.yml

```
@@ -195,8 +193,8 @@ setup:
 
   - do:
       indices.data_streams_stats: {}
-  - match: { _shards.total: 3 }
-  - match: { _shards.successful: 3 }
+  - match: { _shards.total: 6 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)